Source/WebKit/Shared/EntryPointUtilities/Cocoa/XPCService/XPCServiceMain.mm

```diff
@@ -243,6 +243,7 @@ int XPCServiceMain(int, const char**)
             }
         }
 #endif
+        WTF::initialize(); // We use WTF strings.
```

Comment:
This should probably be removed.  The WTF::String use in the following few lines should probably be changed to use NSString and c strings instead.  It should be pretty trivial.

Reply:
Done, will upload shortly